source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -13,9 +13,9 @@ Retrieve Data
 Overview
 --------
 
-You can perform find operations to retrieve data from your MongoDB database.
-You can perform a find operation to match documents on a set of criteria
-by calling the ``Find()`` method.
+In this guide, you can learn how to use the {+driver-long+} to retrieve
+data from your MongoDB collections by using read operations. Call the
+``Find()`` method to match documents on a set of criteria.
```

Comment:
Suggestion:

While I prefer "retrieve" to "match", I think either is ok.
However, I think the preposition "on" isn't commonly used with "match". I think "with", "to" and "by" are more commonly used.